tests/features/bootstrap/ServiceProviderContext.php

```diff
@@ -166,14 +167,48 @@ public function iStartAnSFOAuthentication($nameId)
             $this->loadPrivateKey(new PrivateKey('/var/www/ci/certificates/sp.pem', 'default'))
         );
         $authnRequest->setRequestedAuthnContext(
-            ['AuthnContextClassRef' => ['http://stepup.example.com/assurance/sfo-level2']]
+            ['AuthnContextClassRef' => ['http://stepup.example.com/assurance/loa-self-asserted']]
```

Comment:
Does the name of this method reflect the authnclassref used? Or is this deliberate?